STYLE-GUIDE.md

```diff
@@ -2,38 +2,36 @@
 
 This document defines some general styles we adhere to in the docs.
 
-## Words are important
+## Use Inclusive Language
 
-- Instead of _guys_ try _folks_, _yall_, or _everyone_
-- Instead of _crazy_ try _wild_
-- Instead of _click_ use _select_
-- Instead of _dummy_ use _placeholder_
-- Avoid using gendered language
-- Avoid using violent language (e.g., _kill_)
-- Avoid pop-culture references
+* Instead of _crazy_ try _wild_
+* Instead of _click_ use _select_
+* Instead of _dummy_ use _placeholder_
+* Avoid unnecessarily gendered language: Instead of _guys_ try _folks_, _yall_, or _everyone_.
+* Avoid using violent language (e.g., _kill_)
+* Avoid pop-culture references as such references may not be familiar to all readers.
 
 Want to learn about why these words should be avoided?
 Check out [Google's inclusive documentation guide](https://developers.google.com/style/inclusive-documentation).
 
 ## Headings
 
-- You should be able to scan the headings of a document and get a good sense of the page
-- Every page should have exactly one h1
-- Don't skip heading levels in Markdown files, e.g., if your previous heading level was a 2, the next heading should be a 2 or a 3, but not a 4 or 5
+* Readers should be able to scan the headings of a page and get an effective overview of the page's content.
+* Every page should have exactly one h1.
+* Headings levels should only increment one level at a time.  E.g., if your previous heading level was an h2, the next heading should be a h2 or an h3, but not an h4 or h5.
 
 ## Links
 
-- Link text should be descriptive and be understandable outside of the context that it is in.
-- Avoid link text like _here_, _click here_, _see here_
+* Link text should be descriptive and have meaning outside of the surrounding context: Avoid link text like _here_, _click here_, _see here_ and instead link to the title of the linked page, e.g. "see [Pulumi Pricing](https://www.pulumi.com/pricing/)".  (This is recommended in order to accommodate visually impared users who use screen readers and often jump through the links of a document.)
```

Comment:
sooo our page titles are not good enough right now to require this, so id prefer to come back and add that direction after they get improved

and this is not only for visually impaired folks, its for everyone!